widget/ui/src/components/ListItemButton/ListItemButton.tsx

```diff
@@ -2,24 +2,16 @@ import type { ListItemButtonProps } from './ListItemButton.types';
 
 import React from 'react';
 
-import { ListItem } from '../ListItem/ListItem';
+import { ListItem } from '../ListItem';
 
 import { BaseListItemButton } from './ListItemButton.styles';
 
 function ListItemButton(props: ListItemButtonProps) {
-  const { onClick, id, style, hasDivider, selected, ...restProps } = props;
-  const onClickWithKey = () => {
-    if (onClick) {
-      onClick(id);
-    }
-  };
+  const { onClick, id, ...restProps } = props;
+
   return (
```

Comment:
The whole idea here was ListItemButton is a specific type of ListItem, But here and in token/chains problem PR, the component has changed and it includes ListItem as a part of this component. I mean ListItemButton and ListItem should be used at the same, One of them should do the work.